tests/custom-logger.test.js

```diff
@@ -48,11 +49,18 @@ test('Test generation custom logger default options', assert => {
         'email',
         'password',
         'username',
-        '[*].email',
-        '[*].password',
-        '[*].username',
+        'debugInfo.email',
+        'debugInfo.password',
+        'debugInfo.username',
+        'debugInfo[*].email',
+        'debugInfo[*].password',
+        'debugInfo[*].username',
```

Comment:
this changes may be considered as breaking...

Reply:
It can be. I have to share something internally first, on its feedback we can roll this back or make a v2 of lc39